.github/workflows/publish.yml

```diff
@@ -138,7 +138,8 @@ jobs:
       - name: Dotnet Pack
         shell: bash
         run: |
-          dotnet pack /p:Version=${{ steps.get_version.outputs.version }} --configuration=Release --output=./packages \
+          dotnet pack /p:Version=${{ steps.get_version.outputs.version }} --configuration=Release \
+            /p:PublishDir=./packages
```

Comment:
you need to add `\` at the end of the line here

Reply:
Fixed 👍